### PR TITLE
Enum

### PR DESCRIPTION
**Pull request by chloeyutianyi**

### CODE REVIEW

**Comment by Fleker** [comment]
It looks like you revised a few of the lint issues, but a few of them remain.

File: peripherals/example.yaml [hunk 1]
```diff
@@ -29,6 +29,9 @@ info:
   version: 0.1.0
   'x-foo': 'x-bar'
 
+imports:
```
Comment (Fleker) [inline peripherals/example.yaml:32]: It seems like this change may cause unit tests to fail as Emboss is currently not in the CI environment. If we revert this change and put together a separate `peripherals/example-emboss.yaml` file it should not cause any CI failures.

File: peripherals-yaml/example.yaml [hunk 2]
```diff
@@ -29,6 +29,9 @@ info:
   version: 0.1.0
   'x-foo': 'x-bar'
 
+imports:
```
Comment (Fleker) [inline peripherals-yaml/example.yaml:32]: It looks like you moved this file, but we still require the original. Can you copy this file back and make sure `peripherals/example.yaml` does not include the `imports`?

**Comment by chloeyutianyi** [comment]
@googlebot I fixed it

**Comment by chloeyutianyi** [comment]
@googlebot I fixed it..

**Pull request closed without merging**